demos/cloud-shell-vnet/main.bicep

```diff
@@ -67,10 +71,46 @@ resource containerSubnet 'Microsoft.Network/virtualNetworks/subnets@2021-08-01'
         }
       }
     ]
+    networkSecurityGroup: {
+      id: nsgResourceId
+    }
+  }
+  dependsOn: [
+    networkSecurityGroup
+  ]
+}
+
+resource networkSecurityGroupDefaultRules 'Microsoft.Network/networkSecurityGroups/defaultSecurityRules@2023-05-01' existing = {
+  name: nsgName
```

Comment:
Is this name correct?

Reply:
@hima-gadupudi Can you respond to these suggestions and retest if you make changes?

Reply:
name is correct.